Backpack-SwiftUI/BannerAlert/Classes/BPKBannerAlertType.swift

```diff
@@ -0,0 +1,66 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+extension BPKBannerAlert {
+    /// The type of the banner, which controls the default icon to display and its color.
+    public enum AlertType {
+        case info(accessibilityLabel: String? = nil)
```

Comment:
It may be, but just checking, is this expected to be optional? should we not always ask for this so that all a11y users are ensured to get context?
I guess it's not that bad to have it as optional as it will usually be the case that the banner shall have enough context, but just ckecking